sqlglot/dialects/mysql.py

```diff
@@ -563,9 +563,16 @@ class Generator(generator.Generator):
             exp.DataType.Type.UTINYINT: "TINYINT",
         }
 
+        TIMESTAMP_TYPE_MAPPING = {
+            exp.DataType.Type.TIMESTAMP: "DATETIME",
+            exp.DataType.Type.TIMESTAMPTZ: "TIMESTAMP",
+            exp.DataType.Type.TIMESTAMPLTZ: "TIMESTAMP",
```

Comment:
If this isn't used anymore then we should probably move these inline in `TYPE_MAPPING`